### PR TITLE
feat: add issue template

### PR DESCRIPTION
**Pull request by EncryptedDev**
**Describe the changes in this pull request:**
Added so we can start the transition back to issues!

### CODE REVIEW

**Comment by EncryptedDev** [comment]
@ImRodry approve strings

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    validations:
+      required: true
+  - type: textarea
+    validations:
+      required: true
+    attributes:
+      label: Description
+      description:
+        Describe in detail what would you like to see added to the presence and
+        why.
```
Comment (ImRodry) [inline .github/ISSUE_TEMPLATE/feature_request.yml:22]: why

File: .github/ISSUE_TEMPLATE/service_request.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
+name: Service Request
+description: Request a new presence to be added to the store
+title: Service Name | Service URL
+labels: [Service Request]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Before submitting a new request, ensure your service hasn't already been
+        submitted before. If it has, reply to the open issue instead.
```
Comment (ImRodry) [inline .github/ISSUE_TEMPLATE/service_request.yml:10]:
```suggestion
        Before submitting a new request, ensure your service hasn't already been
        requested before. If it has, reply to the open issue instead.
```

**Review by ImRodry — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/feature_request.yml (hunk 1 above)
Comment (EncryptedDev) [reply .github/ISSUE_TEMPLATE/feature_request.yml:22]: why yourself

**Review by Bas950 — APPROVED** [review]

**Pull request merged**